subhub/api/webhooks/stripeChargeSucceededProcessor.py

```diff
@@ -0,0 +1,8 @@
+from subhub.api.webhooks.abstractStripeWebhookProcessor import AbstractStripeWebhookProcessor
```

Comment:
remove stubbed code that we are not using

Reply:
what stub code?

Reply:
https://github.com/mozilla/subhub/blob/feat/SS-167/subhub/api/webhooks/stripeChargeDisputeFundsWithdrawnProcessor.py

You have several files that do nothing other than print in the run function.  YAGNI: you aint gonna need it; remove these

also consider consolidating your class definitions into one file, especially if they are only going to be several lines each.